CMakeLists.txt

```diff
@@ -24,9 +24,10 @@ set(CATKIN_DEPENDENCIES
 
 set(LIBRARIES
   IROCFleetManager
-  IROC_CoverageManager
-  IROCAutonomyTestManager
-  CoveragePlannerLib
+  IROCFleetManager_WaypointPlanner
+  IROCFleetManager_CoveragePlanner
```

Comment:
We can also create only one library from them that could be named `IROCFleetManager_BasicPlannerPlugins`. You need to change plugins.xml if you do this.